examples/sv3d/train.sh

```diff
@@ -0,0 +1,4 @@
+python train.py \
+    --model_cfg configs/sampling/sv3d_u.yaml \
+    --train_cfg configs/sv3d_u_train.yaml \
+    --debug
```

Comment:
why debug？

Reply:
This was used for debugging with no files dumped, should not be included here. Removed.